test/bdd/fixtures/agent/docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright IBM Corp, SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+version: '2'
+
+services:
+
+  alice.agent.example.com:
+    container_name: alice.aries.example.com
+    image: ${AGENT_FIXTURE_IMAGE}:${AGENT_FIXUTRE_IMAGE_TAG}
+    environment:
+      - GOPATH=/opt/gopath
```

Comment:
GOPATH should not be needed.

Reply:
This is the environment variable I am passing to container which will be used by script to find project. This isn't conventional $GOPATH